packages/omezarr/src/sliceview/tile-renderer.ts

```diff
@@ -6,8 +6,30 @@ import type { vec2, vec3, vec4 } from '@alleninstitute/vis-geometry';
 import type REGL from 'regl';
 import type { Framebuffer2D } from 'regl';
 
+const vert = /*glsl*/ `
```

Comment:
this vertex shader was duplicated between the two commands in this file, so I just dry'd it up a bit